.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
 .netlify
+bower_components/
```

Comment:
Unless we have a really good reason to support it, I'd suggest we not publish this to Bower. While it's maintained as far as bug fixes go, even the maintainers [try to enourage npm/yarn](https://bower.io/) these days. It's unlikely to get many new features, and is already way behind npm.

Reply:
This makes sense. I worked on projects before where we used both Bower and NPM (with the delineation being that Bower is for front-end dependencies and NPM is for back-end dependencies) but I haven't kept up with any of it. We should definitely remove Bower and maintain everything with NPM.